### PR TITLE
Added GeoLocation Feature

### PR DESCRIPTION
**Pull request by zeezeee96**
~Added Geolocation button for accessing location of user.
~Location Click navigates to weatherdetails page.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,5 +39,9 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
```
Comment (Monfernape) [inline package.json]: I am not sure if they should be here. You see devDependencies exist for development purpose and when application goes live, they are not loaded. And I think you need these icons for your application.

File: src/components/Searchbar.js [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
 import Autocomplete from "@material-ui/lab/Autocomplete";
-
+import { Icon } from "@iconify/react";
+import mapMarker from "@iconify-icons/fa/map-marker";
 export const SearchBar = () => {
   const [query, setQuery] = useState("");
   const [options, setOptions] = useState([]);
+  const [latitude, setlatitude] = useState("");
+  const [longitude, setlongitude] = useState("");
+  const [geoLocationInfo, setGeoLocationInfo] = useState([])
```
Comment (Monfernape) [inline src/components/Searchbar.js]: How about renaming it as location? getGeolocation sounds like a method.

File: src/services/request-service.js [hunk 3]
```diff
@@ -31,3 +37,13 @@ export const getCityDetail =  (id) => {
       }}).then((response) => resolve(response.data[0]))
   })
 };
+export const getLocation =  (latitude ,longitude) => {
+  console.log(longitude, latitude)
```
Comment (Monfernape) [inline src/services/request-service.js]: Cleanup required here. Also, remove consoles across the application

**Review by Monfernape — CHANGES_REQUESTED** [review]

File: src/services/request-service.js (hunk 3 above)
Comment (zeezeee96) [reply src/services/request-service.js]: updated

File: src/components/Searchbar.js (hunk 2 above)
Comment (zeezeee96) [reply src/components/Searchbar.js]: done

**Review by Monfernape — APPROVED** [review]

**Pull request merged**